documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -36,14 +36,25 @@ stringData:
   user: <user> <1>
   password: <password> <2>
   cacert: | <3>
-    <engine_ca_certificate>
+    <ca_certificate>
   thumbprint: <vcenter_fingerprint> <4>
 EOF
 ----
 <1> Specify the vCenter user or the {rhv-short} {manager} user.
 <2> Specify the user password.
-<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
+<3> {rhv-short} only: {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate and see the note below. The {manager} CA certificate can be retrieved at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
 <4> VMware only: Specify the vCenter SHA-1 fingerprint.
++
+[NOTE]
+====
+If the {manager} CA certificate was replaced by a third-party certificate, enter the following in the `metadata` section of the `Secret` manifest:
```

Comment:
also need to enter url in stringData section:
stringData:
    url: <api_end_point>